hles/transformation/src/main/scala/org/broadinstitute/monster/dap/HLESurveyTransformationPipelineBuilder.scala

```diff
@@ -23,29 +23,29 @@ object HLESurveyTransformationPipelineBuilder extends PipelineBuilder[Args] {
     val rawRecords = readRecords(ctx, args)
     val rawEnvRecords = readEnvRecords(ctx, args)
 
-    val dogs = rawRecords.transform("Map Dogs")(_.map(DogTransformations.mapDog))
-    val owners = rawRecords.transform("Map Owners")(_.map(OwnerTransformations.mapOwner))
-    val cancer_conditions =
+    val dogs = rawRecords.transform("Map Dogs")(_.flatMap(DogTransformations.mapDog))
+    val owners = rawRecords.transform("Map Owners")(_.flatMap(OwnerTransformations.mapOwner))
```

Comment:
this change will mean that skipped records simply don't show up, rather than showing up as `null` like they did previously.